pkg/migrations/duplicate_test.go

```diff
@@ -102,47 +102,6 @@ func TestDuplicateStmtBuilderCheckConstraints(t *testing.T) {
 	}
 }
 
-func TestDuplicateStmtBuilderUniqueConstraints(t *testing.T) {
```

Comment:
Is it possible to move these tests rather than remove them? Or test the same SQL generation some other way?

Reply:
Because the SQL generation function will not be used anymore and also is removed by this PR, I see no point in keeping this test. Maybe we can add another test for SQL statements generated in `executeCreateUniqueIndexConcurrently` if it makes sense?